src/Google.Api.Gax/ApiCall.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Google.Protobuf;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    /// <summary>
+    /// Wrapper for matching asynchronous and synchronous API calls to the same underlying method.
+    /// </summary>
+    /// <typeparam name="TRequest">Request type, which must be a protobuf message.</typeparam>
+    /// <typeparam name="TResponse">Response type, which must be a protobuf message.</typeparam>
+    public sealed class ApiCall<TRequest, TResponse>
+        where TRequest : class, IMessage<TRequest>
+        where TResponse : class, IMessage<TResponse>
+    {
+        /// <summary>
+        /// Delegate representing a single asynchronous API call with a request and <see cref="CallSettings"/>.
+        /// </summary>
+        /// <param name="request">Request to make to the service.</param>
+        /// <param name="callSettings">The (optional) settings to apply to this call.</param>
```

Comment:
Rather than "(optional)", I'd say that the value can be null, and what the effect of that is.